### PR TITLE
Sankey dependency wheel - Use colors from palette

### PR DESCRIPTION
**Pull request by BugsBunny338**

### CODE REVIEW

File: sankey_dependency_wheel_plugin/src/dp_sankey_dependency_wheel_plugin/SankeyDependencyWheel.jsx [hunk 1]
```diff
@@ -10,14 +14,14 @@ HighchartsWheel(Highcharts);
...
 
-const indigoColors = ['#14B2E2', '#06C08E', '#E54D42', '#F18600', '#AB56A3', '#F4D522', '#93A1AD', '#6BBFD8', '#B589B1', '#EE8780', '#F1AB54', '#85D1BC'];
-
 const SankeyDependencyWheel = (props) => {
     const {
         LoadingComponent,
         ErrorComponent,
         widget,
     } = props;
+    const colorPalette = useDashboardSelector(selectColorPalette).map(color =>
```
Comment (kandl) [inline sankey_dependency_wheel_plugin/src/dp_sankey_dependency_wheel_plugin/SankeyDependencyWheel.jsx:23]: Should it be memoized?

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: sankey_dependency_wheel_plugin/src/dp_sankey_dependency_wheel_plugin/SankeyDependencyWheel.jsx (hunk 1 above)
Comment (BugsBunny338) [reply sankey_dependency_wheel_plugin/src/dp_sankey_dependency_wheel_plugin/SankeyDependencyWheel.jsx]: Done!

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Review by kandl — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-plugin-examples-gate-PR34

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-plugin-examples-sonarqube-scan-zuul-gate/30/ : SUCCESS in 35s
- https://checklist.intgdc.com/job/compliance/job/gooddata-plugin-examples-merge-compliance-check/31/ : SUCCESS in 16s
- https://checklist.intgdc.com/job/compliance/job/gooddata-plugin-examples-fossa-scanning-zuul/7/ : FAILURE in 3m 45s (non-voting)

**Pull request merged**